### ISSUE TITLE
3.10 breaks ember build with yarn workspaces

### ISSUE THREAD

**Issue by cdgeelen** [sir-dunxalot/ember-tooltips#323]
Pull request https://github.com/sir-dunxalot/ember-tooltips/pull/311 is breaking our ember build with yarn workspaces. To illustrate: we have three yarn workspaces: one with ember apps, one with ember engines and one with ember addons. The latter is used to share code and dependencies between engines and applications.

We are using ember-tooltips in our addon workspace, containing the package as a dependency in package.json. However when running ember build, broccoli concat is looking for popper.js in the node_modules folder in our app workspace, while yarn is installing popper.js in the main node_modules folder.

I believe this can be fixed by removing this.project.root from the app.import, as directly referencing node_modules is sufficient (e.g. `app.import('node_modules/popper.js/dist/umd/popper.js');`). At least it works in our cases. Also check the [Ember guides](https://guides.emberjs.com/release/addons-and-dependencies/managing-dependencies/#toc_globals-provided-by-javascript-assets) where they also use this practice.

Directory structure:
```
workspace-root/
+-- package.json
+-- addons/
|    +-- package.json (containing ember-tooltips as a dependency)
|    +-- ...
+-- engines/
|    +-- package.json
|    +-- ...
+-- applications/
|    +-- package.json
|    +-- ...
```

workspace root package.json content:
```json
{
  "private": true,
  "workspaces": [
    "addons/*",
    "engines/*",
    "applications/*"
  ]
}
```

Build error:
```bash
Build Error (SourceMapConcat)

ENOENT: no such file or directory, open '[home]/[app-workspace]/node_modules/popper.js/dist/umd/popper.js'
```

**Comment by loginovma** [comment]
Thank you, @maxfierke!
This also fixes workspace issue for us. Can you please merge & release that?

**Comment by maxfierke** [comment]
@loginovma you may have noticed the new release already, but just in case you haven't, this was released in 3.1.1
